usmqe_tests/api/conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+import json
+
+
+@pytest.fixture
+def not_found_response():
+    """Generate asserts for ``404 Not found`` response."""
+
+    return {
+        "json": json.loads('{"Error": "Not found"}'),
+        "cookies": None,
+        "ok": False,
+        "reason": 'Not Found',
+        "status": 404}
+
+
+@pytest.fixture
+def unauthorized_response():
+    """Generate asserts for ``401 Unauthorized`` response."""
+
+    return {
+        "json": json.loads('{"Error": "Unauthorized"}'),
+        "cookies": None,
+        "ok": False,
+        "reason": 'Unauthorized',
+        "status": 401}
```

Comment:
Both not_found_response and unauthorized_response does not make much sense as fixtures, this could be part of api.tendrl.common module as constants or the same as here - as some methods.